tests/test_ignore.py

```diff
@@ -0,0 +1,156 @@
+from vulture import core
+
+from . import check, skip_if_not_has_async
+
+
+def check_ignore(code, expected, ignore_names, ignore_decorators):
```

Comment:
Could you please change the argument order? I find it much more intuitive to use `check_ignore(code, ignore_names, ignore_decorators, expected)` (as in my earlier comment).

Reply:
Done! :-)